scripts/flowcell_summary_uppload_LIMS.py

```diff
@@ -42,68 +42,75 @@ def get_run_qcs(fc, lanesobj):
-            if dict(fc.udf.items()).has_key('Flow Cell ID'):
-                if '-' in dict(fc.udf.items())['Flow Cell ID']:
-                    flowcell_name = dict(fc.udf.items())['Flow Cell ID']
-                elif dict(fc.udf.items()).has_key('Flow Cell Position'):
-                    flowcell_name = dict(fc.udf.items())['Flow Cell Position'] + dict(fc.udf.items())['Flow Cell ID'] 
-                key = find_flowcell_from_view(fc_db, flowcell_name)
-                if key:
-                    dbobj = fc_db.get(key)
-                    print dbobj['modification_time']+'  '+key
-                    if delta.days < days:
-                        dbobj["illumina"]["run_summary"] = get_sequencing_info(fc) #located in genologics.lims_utils
-                        dbobj["lims_data"]={}
-                        dbobj["lims_data"]['step_id']=fc.id
-                        dbobj["lims_data"]['container_id']=fc.all_inputs()[0].location[0].id
-                        dbobj["lims_data"]['container_name']=fc.all_inputs()[0].location[0].name
-                        info = save_couchdb_obj(fc_db, dbobj)
-                        logging.info('flowcell %s %s : _id = %s' % (flowcell_name, info, key))
-    elif flowcell is not None:
-        if '-' in flowcell:
-            flowcell_name = flowcell
-            fc = lims.get_processes(type = 'MiSeq Run (MiSeq) 4.0', udf = {'Flow Cell ID' : flowcell_name})[0]
-        else:
-            flowcell_name_short = flowcell[1:len(flowcell)]
-            flowcell_position = flowcell[0]
-            fc = lims.get_processes(type = 'Illumina Sequencing (Illumina SBS) 4.0', udf = {'Flow Cell ID' : flowcell_name_short, 'Flow Cell Position' : flowcell_position})[0]
-            flowcell_name = flowcell_position + flowcell_name_short
-        key = find_flowcell_from_view(fc_db, flowcell_name)
+        db_con = xfc_db if fc_is_hiseqx else fc_db  #this might be not neccesary in near future            
+        key = find_flowcell_from_view(db_con, flowcell_name)
         if key:
-            dbobj = fc_db.get(key)
-            dbobj["illumina"]["run_summary"] = get_sequencing_info(fc)
-            get_run_qcs(fc, dbobj['lanes'])
-            info = save_couchdb_obj(fc_db, dbobj)
-            logging.info('flowcell %s %s : _id = %s' % (flowcell_name, info, key))
-                
+            dbobj = db_con.get(key)
+            logging.info('Fetched DB entry for FC %s with key %s' % (flowcell_name, key))
+            dbobj["lims_data"] = {}
+            dbobj["lims_data"]['step_id'] = fc.id
+            dbobj["lims_data"]['container_id'] = fc.all_inputs()[0].location[0].id
+            dbobj["lims_data"]['container_name'] = fc.all_inputs()[0].location[0].name
+            dbobj["lims_data"]["run_summary"] = get_sequencing_info(fc) #located in genologics.lims_utils
+#            get_run_qcs(fc, dbobj['lanes']) ## it is commented and not removed cause of uncertainity
```

Comment:
commented this out as previously this was called only when the script is called for a particular flowcell (which happen very very rare).